include/net/nrf_cloud.h

```diff
@@ -259,8 +267,8 @@ enum nrf_cloud_topic_type {
 	NRF_CLOUD_TOPIC_BULK,
 	/** Endpoint used to publish binary data to nRF Cloud for certain services.
 	 *  One example is dictionary formatted logs enabled by
-	 *  CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_DICTIONARY, which enables the
-	 *  Zephyr option CONFIG_LOG_DICTIONARY_SUPPORT. Binary data published to
+	 *  @kconfig{CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_DICTIONARY}, which enables the
+	 *  Zephyr option @kconfig{CONFIG_LOG_DICTIONARY_SUPPORT}. Binary data published to
 	 *  this topic should be prefixed by the binary header structure defined in
 	 *  nrf_cloud_codec.h -- struct nrf_cloud_bin_hdr.  A unique format value
```

Comment:
I believe our build has hard time rendering the double hyphen to em dash, as it seems to produce crap, see the preview.
I also guess the line 276 is missing a comma at the end.
![image](https://github.com/nrfconnect/sdk-nrf/assets/73991005/54291ad2-265c-4ea6-9a47-0461a0cb0de6)

Reply:
i will change to single dash?  is that preferred?

comma is not required by C.  do the docs like a comma?

Reply:
I guess single dash would do. I just checked other places in the header file, where the last item in the list also had a comma. Not sure about that.